lib/bulk_processor/csv_processor/file_delete_cleanup_processor.rb

```diff
@@ -0,0 +1,13 @@
+class BulkProcessor
+  class CSVProcessor
+    # A File delete object implementation of the Cleanup role
+    class FileDeleteCleanupProcessor
+      def initialize(row_processors)
+      end
+
+      def start
+        BulkProcessor.config.file_class.new(key).delete
```

Comment:
hmm im not sure we have the key here, we might need to pass it in?

Reply:
ohh good catch.. Key exist with payload. Fixed that to pass payload to the file.